packages/core/error.ts

```diff
@@ -1,3 +1,33 @@
-export class RoutingJSError extends Error {}
-export class RoutingJSAPIError extends RoutingJSError {}
-export class RoutingJSClientError extends RoutingJSError {}
+export interface ErrorProps {
+    status_code: number | undefined
```

Comment:
Just a small thing: we don't have to stick to the Valhalla format, and so since we're in TS/JS, we should probably camelCase instead of snake_case :)

Reply:
Yeah I too find camelCase better